airbyte-integrations/connectors/source-brightpearl/.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
+!Dockerfile
+!main.py
+!source_brightpearl
```

Comment:
Just curious why some of these are left out of the image build? I.e. main.py and source_brightpearl? I see that they get copied in as per the Dockerfile. Is this just because we want them copied in slightly differently than they otherwise would be?